build-system/azure-pipeline.template.yaml

```diff
@@ -16,6 +16,12 @@ jobs:
         submodules: recursive  # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
         persistCredentials: true
       # Linux or macOS
+      - bash: |
+          sudo npm install -g azurite@3.17.1
+          sudo mkdir azurite
+          sudo azurite --silent --location azurite --debug azirute/debug.log &
```

Comment:
Instead of relying on a docker image (Linux only), we'll install azurite directly using NPM